### ISSUE TITLE
org-ref-helm-insert-cite-link open the bibtex file instead of inserting the link to the referenced paper.

### ISSUE THREAD

**Issue by sotte** [syl20bnr/spacemacs#8739]
#### Description :octocat:
org-ref-helm-insert-cite-link open the bibtex file instead of inserting the link to the referenced paper.

#### Reproduction guide :beetle:
- Start Emacs
- open org file
- ,ic and select a paper you want to cite; press return to confirm it
- opens the bib file

*Observed behaviour:* :eyes: :broken_heart:
org-ref-helm-insert-cite-link open the bibtex file instead of inserting the link to the referenced paper.

*Expected behaviour:* :heart: :smile:
org-ref-helm-insert-cite-link should insert the link to the referenced paper.

#### System Info :computer:
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.200.9
- Spacemacs branch: master (rev. 8e1af145)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm auto-completion emacs-lisp bibtex org version-control)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

```elisp
(defun dotspacemacs/user-config ()
  (setq org-ref-default-bibliography '("~/tmp/ref.bib")
        org-ref-pdf-directory "~/tmp/papers/"
        org-ref-bibliography-notes "~/tmp/papers.org"
        )
  )
)
```

**Comment by JP-Ellis** [comment]
I can confirm this issue.  I'll see whether this is an upstream issue with org-ref or whether it is a spacemacs issue.

**Comment by sotte** [comment]
Nice, thanks @JP-Ellis!

As a workaround: `org-ref-insert-cite-with-completion` works, but is not as convenient.

**Comment by JP-Ellis** [comment]
The issue should be fixed upstream thanks to jkitchin/org-ref@0ae7023362bb0f1cd0e8ac88ad8a5e0caa4825ca.

**Comment by sotte** [comment]
It works! Thanks!
